unblob/sandbox.py

```diff
@@ -0,0 +1,118 @@
+import ctypes
+import sys
+import threading
+from pathlib import Path
+from typing import Callable, Iterable, Optional, Type, TypeVar
+
+from structlog import get_logger
+from unblob_native.sandbox import (
+    AccessFS,
```

Comment:
I know this comes from Landlock terminology, but this `AccessFS` name looks so alien to me (is it a file-system?).
I would use `FSAccess`, if possible.

Reply:
This would require a rewrite of `unblob-native` which itself follows the naming convention coming from the landlock Rust crate (see https://github.com/onekey-sec/unblob-native/blob/main/src/sandbox/linux.rs#L2). I would keep things as they are right now.